LightBlue.MultiHost/Configuration/IconHelper.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+
+namespace LightBlue.MultiHost.Configuration
+{
+    public static class IconHelper
+    {
+        public static readonly Dictionary<IconOption, string> IconPaths = new Dictionary<IconOption, string>()
```

Comment:
`readonly` only protects the reference here, the dictionary can still be mutated. We should be more clear by changing the exposed type to `IReadOnlyDictioanry<IconOption, string>`